src/App.jsx

```diff
@@ -14,33 +16,91 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const ALPHABET_SORT_CONDITION = 'alphabet';
+  const LENGTH_SORT_CONDITION = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const [sortCondition, setSortCondition] = useState('');
+  const [visibleGoods, setVisibleGoods] = useState(goodsFromServer);
+  const [isReversed, setIsReversed] = useState(false);
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  useEffect(() => {
+    const sortedGoods = [...visibleGoods];
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+    switch (sortCondition) {
+      case ALPHABET_SORT_CONDITION:
+        sortedGoods.sort((a, b) => a.localeCompare(b));
+        break;
+      case LENGTH_SORT_CONDITION:
+        sortedGoods.sort((a, b) => a.length - b.length);
+        break;
+      default:
+        break;
+    }
+
```

Comment:
You can also make a reverse here. Or move it from the useEffect to a separate function